frigg/webhooks/test_views.py

```diff
@@ -131,6 +131,21 @@ def test_push_delete(self, mock_start_build):
         self.assertStatusCode(response)
         self.assertFalse(mock_start_build.called)
 
+    def test_push_delete_mark_builds_as_not_succeeded(self, mock_start_build):
+        project = Project.objects.get_or_create(owner='frigg', name='frigg-hq', private=False)[0]
+        Build.objects.get_or_create(
+            project=project,
+            branch="frecar/add-worker-host"
+        )
+
+        response = self.post_webhook('delete', fixture='push_delete_branch.json')
+        self.assertStatusCode(response)
+        self.assertFalse(mock_start_build.called)
+
+        project = Project.objects.get_or_create(owner='frigg', name='frigg-hq', private=False)[0]
```

Comment:
Is it better to only use a `get()` here? I guess the test should fail if the project has been deleted :)